docs/advanced-features/react-18/streaming.md

```diff
@@ -68,4 +68,6 @@ Currently, data fetching within `Suspense` boundaries on the server side is not
 
 #### Styling
 
-The Next.js team is working on support for `styled-jsx` and CSS modules in streaming SSR. Stay tuned for updates.
+So far only `styled-jsx` is supported with streaming. The Next.js team is working on support for CSS modules and other CSS solutions in streaming SSR. Stay tuned for updates.
```

Comment:
We should also mention inlined styles and global solutions (global CSS import, Tailwind CSS) here.